aikau/src/main/resources/alfresco/services/CrudService.js

```diff
@@ -89,12 +89,15 @@ define(["dojo/_base/declare",
          });
 
          var noRefresh = lang.getObject("data.noRefresh", false, originalRequestConfig);
-         if (noRefresh === true) {
+         if (noRefresh === true) 
+         {
             // Don't make a refresh request...
-         } else {
+         } 
+         else 
+         {
             // When refreshing, check to see if a pubSubScope was provided...
-            var pubSubScope = lang.getObject("data.pubSubScope", false, originalRequestConfig) || "";
-            this.alfPublish(pubSubScope + "ALF_DOCLIST_RELOAD_DATA");
+            var pubSubScope = lang.getObject("responseScope", false, originalRequestConfig) || "";
+            this.alfPublish("ALF_DOCLIST_RELOAD_DATA", null, false, false, pubSubScope);
          }
```

Comment:
I've changed the code, but we have to use responseScope and not alfResponseScope because alfResponseScope should only be set by the Core... the general practice agreed was that when publishing we should only set responseScope and not alfResponseScope... as it was, I tried initially with alfResponseScope (from copying your suggestion exactly) and the unit test failed.